<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>绑定端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBindPort">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>1200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>目标地址:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboTargetIP"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>目标端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinTargetPort">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>3355</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="editMsg"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>发送消息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBroadcast">
       <property name="text">
        <string>广播消息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="text">
    <string>绑定端口</string>
   </property>
   <property name="toolTip">
    <string>绑定端口</string>
   </property>
  </action>
  <action name="actStop">
   <property name="text">
    <string>解除绑定</string>
   </property>
   <property name="toolTip">
    <string>解除绑定</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>清空文本框</string>
   </property>
   <property name="toolTip">
    <string>清空文本框</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
